pkgs/development/compilers/gcc-arm-embedded/10/default.nix

```diff
@@ -1,27 +1,27 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchurl
 , ncurses5
 , python27
```

Comment:
Can we use python3 in the meanwhile?

Reply:
Unfortunately no, the binaries are linked against `libpython2.7.so.1.0`